npm/pkg/dataplane/dataplane.go

```diff
@@ -23,6 +24,24 @@ type Config struct {
 	*policies.PolicyManagerCfg
 }
 
+type updatePodCache struct {
```

Comment:
Move both updatepodcache and endpointcache to types.go and have Get, Set functions which take care of lock unlock inside.

Reply:
discussed offline that this is irrelevant now